customer-commons/src/main/java/no/unit/nva/customer/model/Customer.java

```diff
@@ -0,0 +1,240 @@
+package no.unit.nva.customer.model;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import nva.commons.utils.JacocoGenerated;
+
+import java.time.Instant;
+import java.util.Objects;
+import java.util.UUID;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class Customer {
+
+    public static final String DYNAMODB_KEY_DELIMITER = "#";
+    public static final String IDENTIFIER = "identifier";
+
+    private UUID identifier;
+    private Instant createdDate;
+    private Instant modifiedDate;
+    private String name;
+    private String displayName;
+    private String shortName;
+    private String archiveName;
+    private String cname;
+    private String institutionDns;
+    private String administrationId;
+    private String feideOrganizationId;
+
+    public Customer() {
+    }
+
+    private Customer(Builder builder) {
+        setIdentifier(builder.identifier);
+        setCreatedDate(builder.createdDate);
+        setModifiedDate(builder.modifiedDate);
+        setName(builder.name);
+        setDisplayName(builder.displayName);
+        setShortName(builder.shortName);
+        setArchiveName(builder.archiveName);
+        setCname(builder.cname);
+        setInstitutionDns(builder.institutionDns);
+        setAdministrationId(builder.administrationId);
+        setFeideOrganizationId(builder.feideOrganizationId);
+    }
+
+    public UUID getIdentifier() {
```

Comment:
We could add the @JacocoGenerated tag for not having to test these methods, since they do not do much